packages/block-library/src/post-comment-avatar/edit.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	InspectorControls,
+	__experimentalUseBorderProps as useBorderProps,
+	__experimentalImageSizeControl as ImageSizeControl,
+	useBlockProps,
+} from '@wordpress/block-editor';
+import { PanelBody, ResizableBox } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __, isRTL } from '@wordpress/i18n';
+
+export default ( { attributes, context, setAttributes } ) => {
+	const { className, style, height, width } = attributes;
+	const { commentId } = context;
+
+	// Maximum provided by the WordPress site.
+	const maxImageWidth = 96;
+	const maxImageHeight = 96;
+	if ( ! height || ! width ) {
```

Comment:
This has side effects and we should use `useEffect` for something like that. Having said that, what you need here is to set default values which is possible in `block.json` and `default` prop in the attribute you want.

Reply:
I used `block.json` defined default values. Let me know if that is a better approach.